<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCustomerWindow</class>
 <widget class="QDialog" name="NewCustomerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello newbie!</string>
  </property>
  <widget class="QLabel" name="Label_Name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Age">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Subscribe">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Subscribe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Abort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Surname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>SURNAME</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="Male">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Female">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>New Customer form</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AgeError">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="GenderError">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Comic Sans MS&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
